### PR TITLE
dotnet-tool: add page

### PR DESCRIPTION
**Pull request by navarroaxel**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):** 5.0.405

### CODE REVIEW

**Review by CleanMachine1 — APPROVED** [review]

File: pages/common/dotnet-tool.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# dotnet tool
+
+> Manage .NET tools and search published tools in NuGet.
+> More information: <https://docs.microsoft.com/ef/core/cli/dotnet>.
+
+- Install Entity Framework Core as a global tool:
+
+`dotnet tool install --global dotnet-ef`
```
Comment (EmilySeville7cfg) [inline pages/common/dotnet-tool.md:8]:
```suggestion
- Install a global tool (don't use `--global` for local tools):

`dotnet tool install --global {{dotnetsay}}`
```

Maybe it's better to use some sample global tool and more general description.

File: pages/common/dotnet-tool.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+- Install tools defined in the local tool manifest:
+
+`dotnet tool restore`
+
+- Update a specific global tool:
+
+`dotnet tool update --global {{tool_name}}`
```
Comment (EmilySeville7cfg) [inline pages/common/dotnet-tool.md:16]:
```suggestion
- Update a specific global tool (don't use `--global` for local tools):

`dotnet tool update --global {{tool_name}}`
```

File: pages/common/dotnet-tool.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+- Uninstall a local tool (use `--global` for global tools):
+
+`dotnet tool uninstall {{tool_name}}`
```
Comment (EmilySeville7cfg) [inline pages/common/dotnet-tool.md:20]:
```suggestion
- Uninstall a global tool (don't use `--global` for local tools):

`dotnet tool uninstall --global {{tool_name}}`
```

File: pages/common/dotnet-tool.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+- List installed local tools (use `--global` for global tools):
+
+`dotnet tool list`
```
Comment (EmilySeville7cfg) [inline pages/common/dotnet-tool.md:24]:
```suggestion
- List installed global tools (don't use `--global` for local tools):

`dotnet tool list --global`
```

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]

**Review by EmilySeville7cfg — APPROVED** [review]

**Pull request merged**